module/metrics/compliance.go

```diff
@@ -20,12 +20,28 @@ type ComplianceCollector struct {
 	lastBlockFinalizedAt     time.Time
 	finalizedBlocksPerSecond prometheus.Summary
 	committedEpochFinalView  prometheus.Gauge
+	currentEpochCounter      prometheus.Gauge
+	currentEpochPhase        prometheus.Gauge
 }
 
 func NewComplianceCollector() *ComplianceCollector {
 
 	cc := &ComplianceCollector{
 
+		currentEpochCounter: promauto.NewGauge(prometheus.GaugeOpts{
+			Name:      "consensus_compliance_current_epoch_counter",
```

Comment:
```suggestion
			Name:      "current_epoch_counter",
```

The namespace and subsystem get prepended automatically.